src/sql/workbench/browser/modal/modal.ts

```diff
@@ -479,6 +479,7 @@ export abstract class Modal extends Disposable implements IThemable {
 					this.handleForwardTab(e);
 				}
 			}
+			DOM.EventHelper.stop(e, false);
```

Comment:
Don't do it for all events - we want any events we don't handle to still get propagated up (for things like showing the command palette)